minetest.conf.example

```diff
@@ -1459,6 +1459,102 @@
 #    type: noise_params
 # mgv7_np_cave2 = 0, 12, (67, 67, 67), 10325, 3, 0.5, 2.0
 
+#### Mapgen carpathian
+
+#    Map generation attributes specific to Mapgen Carpathian.
+#    Flags that are not specified in the flag string are not modified from the default.
+#    Flags starting with 'no' are used to explicitly disable them.
+#    type: flags possible values: caverns, nocaverns
+# mgcarpathian_spflags = caverns
+
+#    Y-level of the terrain gradient center point. 
```

Comment:
Needs to be easier to undersand as the parameter for shifting all terrain vertically unchanged.
Also, maybe 'Y-level of lowlands' or something similar?.